src/fluxgui/settings.py

```diff
@@ -57,24 +59,23 @@ def temperature_to_key(temperature):
 
 ################################################################
 
+
 class Settings(object):
 
     def __init__(self):
-        # You can use 'gconftool --dump /apps/fluxgui' to see current
-        # settings on command line.
-        self.client = GConfClient("/apps/fluxgui")
+        self.settings = Gio.Settings.new('apps.fluxgui')
```

Comment:
What's the new way to dump the settings?

Reply:
> What's the new way to dump the settings?

The `Gio.Settings` class already provid the methods for getting and setting the settings(e.g. `get_string`, `set_string`) take a look [here](https://lazka.github.io/pgi-docs/Gio-2.0/classes/Settings.html)

Reply:
Right, but for debugging I've found it useful to be able to dump the settings on the command line. Do you know what the new command is, or if `gconftool --dump /apps/fluxgui` will still work? In any case, I'd like to have the way to dump the settings on the command line written down somewhere.

Reply:
Looks like the new command might be `gsettings list-recursively apps.fluxgui`, but I haven't installed the schemas yet ..

Reply:
For locally installed schemas, by running `glib-compile-schemas .` in the repo, you can view the schema with `gsettings --schemadir . list-recursively apps.fluxgui`. But I'm not sure this is going to show user changes to the defaults, e.g. when the user sets the zip code.

Reply:
> For locally installed schemas, by running `glib-compile-schemas .` in the repo, you can view the schema with `gsettings --schemadir . list-recursively apps.fluxgui`. But I'm not sure this is going to show user changes to the defaults, e.g. when the user sets the zip code.

I tested it here, it works.

Reply:
Great, I added a comment documenting this.